Zend/Optimizer/dfa_pass.c

```diff
@@ -410,7 +410,7 @@ int zend_dfa_optimize_calls(zend_op_array *op_array, zend_ssa *ssa)
 				 && (send_needly->opcode == ZEND_SEND_VAL
 				  || send_needly->opcode == ZEND_SEND_VAR)
 			    ) {
-					int ok = 1;
+					bool ok = 1;
 
```

Comment:
Doesn't the name of the variable `ok` indicate that it is always true? Also can we not use `TRUE` instead of `1`?

Reply:
No, if you look below this can be assigned to 0. I'm not really here to rename vars as I don't totally understand the whole code block.